browser/updater/features.cc

```diff
@@ -18,8 +26,22 @@ BASE_FEATURE(kBraveUseOmaha4Alpha,
              "BraveUseOmaha4Alpha",
              base::FEATURE_DISABLED_BY_DEFAULT);
 
+bool ShouldUseOmaha4(base::Time now, std::optional<bool>& state) {
+  if (!state.has_value()) {
+    int days_since_null = (now - base::Time()).InDays();
+    // Whether Omaha 4 should be used is mostly determined by the feature flag.
+    // However, we also want to give the legacy implementation a chance to run
+    // every 5 days. This lets us recover from a situation where updates with
+    // Omaha 4 are broken because of a bug. Once Omaha 4 is stable, we can
+    // remove the periodic fallback.
+    state = base::FeatureList::IsEnabled(kBraveUseOmaha4Alpha) &&
+            days_since_null % 5;
```

Comment:
`5` can be a feature parameter.

Reply:
Implemented in 7f53ad6fc2872f8664bf1ba9a74681fa3a52d697.